roles/variables/vars/main.yml

```diff
@@ -2025,7 +2047,7 @@ kafka_connect_replicator_properties:
       oauthbearer.jwks.endpoint.url: "{{oauth_jwks_uri}}"
       oauthbearer.sub.claim.name: "{{ oauth_sub_claim }}"
       oauthbearer.expected.issuer: "{{ oauth_issuer_url }}"
-  kafka_connect_oauth_scope:
+  kafka_connect_oauth_claim:
     enabled: "{{kafka_connect_replicator_oauth_enabled and oauth_groups_scope!='none' }}"
     properties:
       oauthbearer.groups.claim.name: "{{ oauth_groups_scope }}"
```

Comment:
looks like a bug this should be oauth_sub_claim.

Reply:
I have just renamed to have the variable name in consistent with other components variable name. It provides better readability.

Reply:
Fixed as part of https://github.com/confluentinc/cp-ansible/pull/1843